apps/pie-storybook/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "pie-storybook",
+  "version": "0.1.0",
```

Comment:
Forgot to mention. I think the `version` can be removed. I don't think we'll be wanting to publish the storybook instance as it won't be used by consumers 👍🏼

Reply:
We don't need to publish Storybook as a package, however not having a version field seems to be a [bad practice](https://docs.npmjs.com/creating-a-package-json-file#:~:text=A%20package.json-,file%20must%20contain,-%22name%22%20and)

> A package.json file must contain "name" and "version" fields.

Reply:
Cool, sounds good! Yeah that's fine with me. This will still continue to play along nicely with changesets 👍🏼